src/desktop/apps/artwork/client/index.coffee

```diff
@@ -165,6 +165,14 @@ module.exports =
     splitTest('artwork_sidebar_pageviews').view() if CLIENT.pageviews?
     exec sharedInit
 
+    hasIntercomLabFeature = CurrentUser.orNull()?.hasLabFeature('Intercom on BMMO Works')
```

Comment:
Is this lab feature name a typo?

Reply:
oops, yea i created it on staging w/ the typo! 🙈